core/cmd/remote_client.go

```diff
@@ -1192,6 +1193,67 @@ func normalizePassword(password string) string {
 	return url.PathEscape(strings.TrimSpace(password))
 }
 
+// SetLogLevel sets the log level on the node
+func (cli *Client) SetLogLevel(c *clipkg.Context) (err error) {
+	if !c.Bool("level") {
```

Comment:
This should be a string

Reply:
Suggests that we need a test in `remote_client_test.go` for these endpoints. Just something very light as it looks like we do most of the testing in the controller layer

Reply:
had some flakiness when writing individual tests and was getting inconsistent results on each run on ci, possibly because hitting the same endpoint, couldn't reproduce locally so I consolidated into one test and it seemed to solve the problem